stix_shifter_modules/nozomi/configuration/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "connection": {
+    "type": {
+      "displayName": "Nozomi",
```

Comment:
Nozomi is the company name and not a product name. It would make more sense to set the display name as the specific product this connector is communicating with.

Additionally the module name should be changed as well to match the product name, not the company name.

Reply:
We are working on this will update you

Reply:
Changed the module name to nozomi_vantage